content/en/account_management/faq/api-app-key-management.md

```diff
@@ -26,7 +35,7 @@ To add a Datadog API or application key, navigate to [Integration -> APIs][1], e
 
 ## Remove
 
-To remove a Datadog API or application key, navigate to [Integration -> APIs][1] and select the **Revoke** button next to the key you want to remove:
+To remove a Datadog API or application or public API key, navigate to [Integration -> APIs][1] and select the **Revoke** button next to the key you want to remove:
```

Comment:
"To remove a Datadog API or application or public API key" should be "Datadog API, application, or public API key"